db/manual_testing_fixtures/key_performance_indicators/assessment_status.rb

```diff
@@ -0,0 +1,37 @@
+require_relative 'setup.rb'
```

Comment:
Can you move all this stuff to `db/dev_fixtures/key_performance_indicators`? I'd like to consolidate